composer/functions/composer_storage_trigger.py

```diff
@@ -46,16 +49,22 @@ def trigger_dag(data, context=None):
     webserver_id = 'YOUR-TENANT-PROJECT'
     # The name of the DAG you wish to trigger
     dag_name = 'composer_sample_trigger_response_dag'
+
+    if USE_EXPERIMENTAL_API:
+        endpoint = 'api/experimental/dags/{}/dag_runs'.format(dag_name)
```

Comment:
Should these use f-strings?

Reply:
probably. I'll test them with f strings now just to check

Reply:
```suggestion
        endpoint = f'api/experimental/dags/{dag_name}/dag_runs'
```